src/main/java/at/hannibal2/skyhanni/features/mining/OreBlock.kt

```diff
@@ -222,7 +222,7 @@ enum class OreBlock(
         checkArea = { inGlacite },
     ),
     LOW_TIER_TUNGSTEN(
-        checkBlock = { it.block == Blocks.cobblestone },
+        checkBlock = { it.block.let { it == Blocks.cobblestone || it == Blocks.monster_egg } },
```

Comment:
You should also check for the monster egg block variant, since this just triggers on all monster egg blocks (stone, cobble, bricks,...)